common/lib/xmodule/xmodule/js/spec/video/sjson_spec.js

```diff
@@ -23,12 +25,43 @@ function (Sjson) {
         });
 
         it('search returns a correct caption index', function () {
-            expect(sjson.search(0)).toEqual(-1);
-            expect(sjson.search(3120)).toEqual(1);
-            expect(sjson.search(6270)).toEqual(2);
-            expect(sjson.search(8490)).toEqual(2);
-            expect(sjson.search(21620)).toEqual(4);
-            expect(sjson.search(24920)).toEqual(5);
+            expect(sjson.search(videoStops[0])).toEqual(0);
+            expect(sjson.search(videoStops[1])).toEqual(1);
+            expect(sjson.search(videoStops[2])).toEqual(2);
+            expect(sjson.search(videoStops[3])).toEqual(2);
+            expect(sjson.search(videoStops[4])).toEqual(4);
+            expect(sjson.search(videoStops[5])).toEqual(5);
+        });
+
+        it('search returns the last entry for a value outside the bounds of the array', function() {
+            expect(sjson.search(OUT_OF_BOUNDS_STOP)).toEqual(sjson.getCaptions().length - 1);
+        });
+
+        it('search returns the first entry for a negative index in the array', function() {
+            expect(sjson.search(-1)).toEqual(0);
+        });
+
+
```

Comment:
It looks like you've been using a single empty line to separate tests; can we continue that here?